Parent/Parent/Conversations/ComposeViewController.storyboard

```diff
@@ -16,19 +16,23 @@
                         <rect key="frame" x="0.0" y="0.0" width="375" height="667"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" translatesAutoresizingMaskIntoConstraints="NO" id="Mok-pg-KkL">
+                            <scrollView clipsSubviews="YES" multipleTouchEnabled="YES" contentMode="scaleToFill" ambiguous="YES" translatesAutoresizingMaskIntoConstraints="NO" id="Mok-pg-KkL">
```

Comment:
How hard would it be to clean up the ambiguous constraints in this storyboard?

Reply:
This appears to be an issue on master as well. It just became a thing because I modified the storyboard and it attached all the ambiguous attributes.